torch/_C/__init__.pyi.in

```diff
@@ -621,6 +631,10 @@ class EnumType(JitType):
     ) -> None:
         ...
 
+class TensorType(JitType):
+    @classmethod
+    def get(cls) -> TensorType: ...
```

Comment:
Just curious, why is this not a `staticmethod` with no args?

Reply:
No particular reason. This code was introduced in a different PR. I just moved it to be close to the other methods that you defined before. I can the `@classmethod` to a `@staticmethod` if you want.